### DISCUSSION TITLE
Ability to add any column to the top level logs view

### DISCUSSION THREAD

**Discussion by RoryCrispin** [grafana/clickhouse-datasource#696] [Ideas]
In Explore mode, it's possible to configure which columns make up the log message presented in the main logs view. To do so, click the 'Eye' button on a field on the detail view (pictured)
<img width="1839" alt="Screenshot 2024-02-12 at 16 57 46" src="https://github.com/grafana/clickhouse-datasource/assets/716269/bbd3d718-1320-492b-9155-ab47b41bc755">

This should be possible in the Dashboard view as well.

Workaround: build a fake body string using ClickHouse like:  
`SELECT concat('body: ', Body, ' logger_name: ', logger_name) as Body, Body as Body_raw, logger_name` to

**Comment by SpencerTorres** [comment]
Possibly related: new `Labels` input: #941
